### DISCUSSION TITLE
Why is the received data repeated and how to use SerialPort#flush ？

### DISCUSSION THREAD

**Discussion by steiner0424** [serialport/node-serialport#2288] [Q&A]
### Summary of Problem
I use "port.on('data', function (data) {})" to receive serial data,But every time I use port.close and port.open receive serial data again, There will always be repeated data reception, which will be repeated every time.I want to know if SerialPort flush can solve this problem and how to use it.

### Versions, Operating System and Hardware

- SerialPort@9.06?
- Node.js v14.15.4?
- Windows

**Comment by julietafoglia** [comment]
Did you manage to solve this?
